app/src/handlers/census/MetagameEventEventHandler.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ *  ### CENSUS RESPONSE ATTRIBUTES ####
+ "event_name":"MetagameEvent",
+ "experience_bonus":"",
+ "faction_nc":"",
+ "faction_tr":"",
+ "faction_vs":"",
+ "metagame_event_id":"",
+ "metagame_event_state":"",
+ "timestamp":"",
+ "world_id":"",
+ "zone_id":""
+ * ### END ###
+ **/
+
+import {injectable} from 'inversify';
+import EventHandlerInterface from '../../interfaces/EventHandlerInterface';
+import {GenericEvent} from 'ps2census/dist/client/utils/PS2Events';
+import {getLogger} from '../../logger';
+import config from '../../config';
+import {jsonLogOutput} from '../../utils/json';
+import MetagameEventEvent, {MetagameEventState} from './events/MetagameEventEvent';
```

Comment:
Spaces within brackets

Reply:
I think you checked the outdated sources here?

Reply:
Yeah I think it was, apologies.